Forge.TreeWalker.UnitTests/Forge.TreeWalker.UnitTests.csproj

```diff
@@ -44,29 +44,35 @@
     <PackageReference Include="Newtonsoft.Json.Schema" Version="3.0.11" />
     <PackageReference Include="MSTest.TestAdapter" Version="1.3.2" />
     <PackageReference Include="MSTest.TestFramework" Version="1.3.2" />
+    <PackageReference Include="NUnit">
+      <Version>3.13.2</Version>
+    </PackageReference>
   </ItemGroup>
   <ItemGroup>
     <Reference Include="System" />
     <Reference Include="System.Core" />
+    <Reference Include="System.Runtime.Serialization" />
+    <Reference Include="System.Xml" />
```

Comment:
I try to keep the references as tight as possible in these projects. Please review if any of these additions can be removed.